native/src/modules/app/createNavigateToInternalLink.js

```diff
@@ -22,24 +23,25 @@ export const createNavigateToInternalLink = ({
   navigateToCategory: NavigateToCategoryParamsType => void,
   navigateToDashboard: NavigateToCategoryParamsType => void
 }) => ({ url, language }: NavigateToInternalLinkParamsType) => {
-  const parts = url.split('/').filter(segment => segment)
-  const pathnameParts = parts.splice(2)
+  const parsedUrl = new URL(url)
+  const pathname = parsedUrl.pathname
+  const pathnameParts = pathname.split('/').filter(Boolean)
   const newCity = pathnameParts[0]
   const newLanguage = pathnameParts[1]
-  const pathname = pathnameParts.reduce((acc, part) => `${acc}/${part}`, '')
+  const normalizedPathname = pathnameParts.reduce((acc, part) => `${acc}/${part}`, '')
```

Comment:
Added this as double slashes are actually allowed in URL to current standard. See [here](https://url.spec.whatwg.org/#dom-url-pathname). Alternatively I tried using `react-native-url-polyfill` and I tried using  `ẁhatwg-url`, but as they both also follow the URL standard directly the result did not change. Do we really need a handling for double slashes in URLs?

I do not feel that this PR is a real improvement. Other opinions?

Reply:
I am quite sure that this was not to normalize double slashes, but rather to just have a pathname at all. So this line is not necessary imo, just use the pathname from above. Previously we did not have a pathname at all which is why we had to create it manually by splitting and splicing from the url.
The only improvement should be that we don't have to split the url manually to remove the `https://integreat.app` to get the pathname which is admittedly really a minor improvement.

Reply:
The filter above is probably not needed as well.

Reply:
The filter is needed, as every pathname starts with a `/`. `/augsburg.split('/')` would therefore result in `['', 'augsburg']`